release/build.sh

```diff
@@ -33,17 +33,55 @@ mkdir $output
 cp release/play_abstreet.$ext release/ungap_the_map.$ext release/INSTRUCTIONS.txt $output
 
 for name in game cli fifteen_min osm_viewer parking_mapper santa ltn; do
-	cp target/release/${name}${suffix} $output;
```

Comment:
🚨🚨🚨THERE ARE A MIXTURE OF TABS AND SPACES IN THIS FILE🚨🚨🚨 😱

Reply:
That's the sort of chaos that unfolds when you untether a programmer from a job with regular code review for years! -mad scientist gargling-